easybuild/easyblocks/generic/rpackage.py

```diff
@@ -134,10 +134,19 @@ def make_cmdline_cmd(self, prefix=None):
         else:
             prefix = ''
 
+        loc = self.start_dir
+        if loc is None:
+            loc = self.ext_dir or self.ext_src
```

Comment:
@lexming We should also have a deprecation message for this fallback mechanism?

If a sufficiently recent EasyBuild framework is used, `self.start_dir` should never be `None` anymore?

Reply:
`None` is still ok, it can still be the case for those extensions that are not extracted in the build directory (_e.g._ Python wheels). See the table in https://github.com/easybuilders/easybuild-framework/pull/4206#issuecomment-1419205467